doc/classes/Node.xml

```diff
@@ -115,6 +115,7 @@
 				To consume the input event and stop it propagating further to other nodes, [method Viewport.set_input_as_handled] can be called.
 				This method can be used to handle Unicode character input with [kbd]Alt[/kbd], [kbd]Alt + Ctrl[/kbd], and [kbd]Alt + Shift[/kbd] modifiers, after shortcuts were handled.
 				For gameplay input, this and [method _unhandled_input] are usually a better fit than [method _input] as they allow the GUI to intercept the events first.
+				This method also performs better than [method _unhandled_input], since unrelated events such as [InputEventMouseMotion] are automatically filtered.
```

Comment:
Here one can be more precise: "By default only [InputEventKey] events are propagated to this function.".

Reply:
This is already included in the first line of this method's documentation: 

> Called when an [InputEventKey] or [InputEventShortcut] hasn't been consumed by [method _input] or any GUI [Control] item. The input event propagates up through the node tree until a node consumes it.

Reply:
Makes sense.
The text, you have cited could be improved, because `InputEventShortcut` events are not sent to `_unhandled_key_input`.